app/controllers/miq_ae_class_controller.rb

```diff
@@ -1143,7 +1143,7 @@ def update_method
       begin
         MiqAeMethod.transaction do
           set_input_vars(ae_method)
-          ae_method.inputs.destroy(MiqAeField.find_all_by_id(@edit[:fields_to_delete]))
+          ae_method.inputs.destroy(MiqAeField.where("id = ?", @edit[:fields_to_delete]))
```

Comment:
Hash good; string with placeholder okay; string with direct interpolation (very) bad. Hmm.. maybe we should pull Hash Conditions up above Array Conditions.